subgraphs/aave-v2/src/modules/Price.ts

```diff
@@ -0,0 +1,46 @@
+import * as utils from "../common/utils";
+import * as constants from "../common/constants";
+import { getPriceOracle } from "../common/initializers";
+import { Address, BigDecimal, BigInt, log } from "@graphprotocol/graph-ts";
+
+export function getAssetPriceInUSDC(tokenAddress: Address): BigDecimal {

```

Comment:
If we get 0 or a negative asset price aave has a fallback oracle that we should implement: https://docs.aave.com/developers/v/2.0/the-core-protocol/price-oracle#getfallbackoracle